h2o-genmodel/src/main/java/hex/genmodel/algos/gam/GamMojoMultinomialModel.java

```diff
@@ -0,0 +1,68 @@
+package hex.genmodel.algos.gam;
+
+import java.util.Arrays;
+
+public class GamMojoMultinomialModel extends GamMojoModelBase {
+    private boolean _multinomial; // true multinomial, false ordinal
```

Comment:
Let's choose a better name in order not to require reading the commentary. Something like `trueMultinomial` would do.

Reply:
I use _trueMultinomial